source/stylesheets/base/_navigation.scss

```diff
@@ -12,35 +12,52 @@
   .container {
     display: flex;
     flex-direction: row;
+    position: relative;
 
-    img {
-      transition: height 0.1s ease;
+    .blog-title {
+      color: $red;
+      font-size: 87px;
+      line-height: 80px;
+      margin-bottom: 0.3em;
+      transform: rotate(-8deg);
+      transition: font-size 0.1s ease;
+    }
+
+    .blog-title-first-line {
+      padding-left: 15px;
     }
 
     h3 {
       flex: 1 0 0;
     }
 
     ul {
-      align-content: space-evenly;
-      align-items: flex-end;
-      background: url("../images/nav_background.svg") no-repeat top right;
+      bottom: 20px;
       display: flex;
-      flex: 2 0 0;
-      justify-content: flex-end;
       list-style: none;
+      position: absolute;
+      right: 0;
     }
 
     li {
       a {
+        border-bottom: 0 solid transparent;
         color: $blue;
         font-weight: 400;
-        padding: 3px 20px;
+        padding-bottom: 36px;
+        padding-left: 20px;
+        padding-right: 20px;
         text-decoration: none;
-        &:hover {
-          background: url("../images/link-background.svg") no-repeat center center;
-          background-size: contain;
-          color: $white;
+        transition: border-bottom $base-duration;
+
+        &:hover,
+        .active {
+          border-bottom: 4px solid $blue;
+          padding-bottom: 36px;
+        }
+
+        .active {
```

Comment:
The nesting in this file is borderline. I've hinted at this before, but you should move towards keeping things top level. [Check out this article](http://thesassway.com/intermediate/avoid-nested-selectors-for-more-modular-css) and put it into practice while you're refactoring.